ansible/roles/ansible-tvault-contego-extension/tasks/get_venv.yml

```diff
@@ -1,22 +1,25 @@
 ---
-
-- name: find  venv is using or not
-  shell: "{{ PYTHON_VERSION }}  -c 'import sys; print(sys.prefix)'"
+- name: Find ENVPATH
+  shell: "{{virtual_env}} && {{ PYTHON_VERSION }} -c 'import sys; print(sys.prefix)'"
   register: ENVPATH
 
 - set_fact:
     ENV_PATH : "{{ENVPATH.stdout}}"
 
+- debug:
+    msg: "{{ENV_PATH}}"
+    verbosity: "{{ verbosity_level }}"
+
 - set_fact:
-    RECOMMENDS_VAR : "no"
+    EXTRA_APT_VAR : "--no-install-recommends"
```

Comment:
I don't this its required anymore. check the pkg dependency for deb/rpm if we have mentioned the 'recommends' section.

Reply:
yes, I checked its no longer required, so removed